cmd/envtool/tests.go

```diff
@@ -132,12 +133,26 @@ func runGoTest(ctx context.Context, args []string, total int, logger *zap.Sugare
 			}
 
 			logger.Info(msg)
+			if event.Action == "fail" {
+				for _, item := range outputs {
+					logger.Info(item)
+				}
+			}
+
+			// delete test outputs end of cycle
+			outputs = []string{}
 
 		case "start": // the test binary is about to be executed
 		case "run": // the test has started running
+			// delete test outputs start of cycle
+			outputs = []string{}
```

Comment:
That will not work correctly for parallel tests (see pause, cont below). So let's extend `error_test.go` with some examples and improve this code